src/components/FindMovie/FindMovie.js

```diff
@@ -16,29 +30,34 @@ export const FindMovie = () => (
           <input
             type="text"
             id="movie-title"
+            value={query}
             placeholder="Enter a title to search"
-            className="input is-danger"
+            className={`input ${error ? 'is-danger' : ''}`}
```

Comment:
`classnames` library. Get used to that library. It's very popular and used in the most projects